lib/stdlib/src/gen_event.erl

```diff
@@ -19,99 +19,104 @@
 %%
 -module(gen_event).
 -moduledoc """
-Generic event handling behavior.
+    Generic event handling behavior.
```

Comment:
Why add the extra indentation? We don't have that indentation in other module. It also complicates the review as its makes it's hard to see what you have changed.

Reply:
Indentation removed.  Unfortunately I reflowed much of the text due to rephrasing and link updating so it is still not that easy to see in a diff exactly what I have changed...